components/nimbus/src/persistence.rs

```diff
@@ -227,13 +262,93 @@ impl Database {
     }
 
     fn maybe_upgrade(&self) -> Result<()> {
+        log::debug!("entered maybe upgrade");
         let mut writer = self.rkv.write()?;
         let db_version = self.meta_store.get::<u16, _>(&writer, DB_KEY_DB_VERSION)?;
         match db_version {
             Some(DB_VERSION) => {
                 // Already at the current version, no migration required.
                 return Ok(());
             }
+            Some(1) => {
+                log::debug!("Upgrading from version 1 to version 2");
```

Comment:
This seems like it might be worth upgrading to an `info` to me.  It shouldn't be happening enough to be too chatty, I hope.

Reply:
Done.